site2/website/versioned_docs/version-2.8.0/developing-binary-protocol.md

```diff
@@ -382,6 +382,29 @@ Parameters:
  * `validation_error` → *(optional)* Indicates that the consumer has discarded
    the messages due to: `UncompressedSizeCorruption`,
    `DecompressionError`, `ChecksumMismatch`, `BatchDeSerializeError`
+ * `properties` -> *(optional)* Reserved configuration items
+ * `txnid_most_bits` -> *(optional)* Same as TC ID, `txnid_most_bits` and `txnid_least_bits`
```

Comment:
```suggestion
 * `txnid_most_bits` -> *(optional)* Same as the TC ID. `txnid_most_bits` and `txnid_least_bits`
```
The TC ID means Transaction ID or something else? If yes, it looks you use Transaction ID in the following section. It's better to unify them.

Reply:
Yeah, I will involve the updates of the master branch as well to address them all.